src/Adapter/Presenter/Product/ProductLazyArray.php

```diff
@@ -829,13 +897,15 @@ public function addQuantityInformation(
                 if ($product['quantity'] < $settings->lastRemainingItems) {
                     $this->applyLastItemsInStockDisplayRule();
                 } else {
+                    $config = $this->configuration->get('PS_LABEL_IN_STOCK_PRODUCTS');
                     $this->product['availability_message'] = $product['available_now'] ? $product['available_now']
-                        : Configuration::get('PS_LABEL_IN_STOCK_PRODUCTS', $language->id);
+                        : ($config[$language->id] ?? $config[0] ?? null);
```

Comment:
Same here you should add `()` for less confusion

Reply:
Fixed